examples/bond/package.json

```diff
@@ -20,7 +20,7 @@
     "typescript": "^5.6.3"
   },
   "dependencies": {
-    "@lfdecentralizedtrust-labs/paladin-sdk": "^0.9.0",
+    "@lfdecentralizedtrust-labs/paladin-sdk": "file:../../sdk/typescript",
```

Comment:
I can't remember if we said we'd leave `v0.9.0` for this release and maybe change to something else later?

Reply:
We agreed to leave this change as-is to avoid creating too much disruption in the PR. It has no impact on end users, since they are required to override the SDK version to the latest.